CHANGES.rst

```diff
@@ -192,7 +192,7 @@ Features
   `#4244 <https://pulp.plan.io/issues/4244>`_
```

Comment:
I'm not involved in this project so feel free to disregard, but I don't think we can rewrite the history at all. Those older versions contain those older names no matter what.

Reply:
that's correct. i don't think it depends on the project - changelog is not supposed to be rewritten